lib/dul_argon_skin/configurable.rb

```diff
@@ -8,8 +8,10 @@ module Configurable
       mattr_accessor :request_base_url do
         ENV['REQUEST_BASE_URL'] ||= 'https://requests.library.duke.edu'
       end
+      mattr_accessor :request_base_url do
```

Comment:
This mattr_accessor needs a unique name -- like map_location_service_url: `mattr_accessor :map_location_service_url do`

Reply:
oops! Thanks for catching that :)